test/suites/common-all/fee_balance_transfer/test_fee_balance_transfer.ts

```diff
@@ -23,6 +24,8 @@ describeSuite({
             bob = context.keyring.bob;
             polkadotJs = context.polkadotJs();
             baseWeight = extractWeight(polkadotJs.consts.system.blockWeights.perClass.normal.baseExtrinsic).toBigInt();
+            const runtimeName = polkadotJs.runtimeVersion.specName.toString();
+            isRelay = runtimeName.includes("light");
```

Comment:
Was this on purpose? Should be the same but just asking.
```suggestion
            isRelay = runtimeName.includes("starlight");
```

Reply:
because in the future we might have "starlight" "dancelight" "stagelight"...just preventing changes in the future

Reply:
ideally common-x should serve to test all those, hence the reason to include just "light"

Reply:
dsregard